node-launchpad/src/components/status.rs

```diff
@@ -417,6 +426,7 @@ impl Component for Status {
                     }
 
                     if self.lock_registry.is_some() {
+                        // FIXME: What do we show to the user?
```

Comment:
When `lock_registry.is_some()`, we've got the pop up saying that we're starting/stopping nodes. Do you mean that we should display something else instead?

Reply:
You are right. My mistake. This prevents multiple execution of this part of the code. I'll remove the `FIXME`.